### ISSUE TITLE
Idea: "Boot Asahi" App

### ISSUE THREAD

**Issue by alessaba** [AsahiLinux/asahi-installer#340]
I recently installed Asahi on my M1 Air and was looking for a quicker way to boot the Linux partition

I created a small app (basically a wrapper to a shell script) that just asks for your fingerprint/password (to run as root), calls bless to set the next-boot disk to Asahi, then reboots automatially to Asahi It's a very simple app. If you want to try you can 
[download it here](https://github.com/user-attachments/files/17803314/Boot.Asahi.zip). If you don't trust the code, you can open the executable with TextEdit, but here is the spoilers for what code actually runs

`#!/bin/sh

sudo bless --mount /Volumes/Asahi --setBoot --nextonly
sudo reboot`

The reason why i am opening the issue is not only to share this very simple idea, but to propose the addition of such app with the Asahi Installer. It should install the app before asking to reboot. And when the user needs to boot back to Linux, it's just as simple as opening the app. (no more Shutdown + long press the power button!)

Let me know what you think

**Comment by alessaba** [comment]
Update: this version uses a simple AppleScript to do the same thing
[Boot Asahi.app.zip](https://github.com/user-attachments/files/17819992/Boot.Asahi.app.zip)

**Comment by chadmed** [comment]
You can just use the Startup Disk UI that's already built in to macOS.

**Comment by alessaba** [comment]
Yeah, that's not a one-click solution like i proposed, but that's fair. The small script to do it is simple anyways, if someone wants to do it on their machine, it will be easy when they find this issue :-)
